### ISSUE TITLE
It is not possible to enter the letters Ś and ś in Polish language.

### ISSUE THREAD

**Issue by Slawek234** [openstreetmap/iD#4834]
It is not possible during editing to enter the letters Ś and ś in Polish language.
By choosing the alt(right)+s combination instead of the letter ś, the ID editor tries to save changes.

**Comment by maraf24** [comment]
Pressing Right Alt + S on "polish programmer keyboard" layout - standard layout in Poland - generates Ctrl-Alt-S keyboard event.  
In application - here iD - this event must be unhandled for the letter Ś to appear in edit field.
And this is where problem lies: you can invoke saving changes in iD by pressing Ctrl-Alt-S, Ctrl-Shift-S and Ctrl-S. However only the last shortcut is officially supported.
In order to fix the problem, we need Ctrl-Alt-S to stop working as a keyboard shortcut.

**Comment by bhousel** [comment]
> And this is where problem lies: you can invoke saving changes in iD by pressing Ctrl-Alt-S, Ctrl-Shift-S and Ctrl-S. However only the last shortcut is officially supported.
> In order to fix the problem, we need Ctrl-Alt-S to stop working as a keyboard shortcut.

Ok I think I understand how to fix this.  We'd need to make sure _not_ to match the shortcut if Alt is pressed.

Just to make sure, is Ctrl-S a normal shortcut to use for a Save command in a Polish localization?
In en-US, it's pretty much universal that Ctrl-S means "save", but I wouldn't want to assume that this is true for Polish too:

<img width="200" alt="screenshot 2018-05-04 16 57 31" src="https://user-images.githubusercontent.com/38784/39652719-7e8de964-4fbc-11e8-9c6c-59a811a23e50.png">

**Comment by maraf24** [comment]
> Just to make sure, is Ctrl-S a normal shortcut to use for a Save command in a Polish localization?

Yes it is. Even though 'Save' is "Zapisz' in Polish, It wasn't changed as Ctrl+S is a standard shortcut in all Polish-localized software for saving data, probably thanks to popularity of Microsoft Office.

**Comment by Slawek234** [comment]
In all Polish localization ctrl + s is used for saving (zapisz) <a href='https://postimg.cc/image/5vglwa3i5/' target='_blank'><img src='https://s14.postimg.cc/5vglwa3i5/ssss.jpg' border='0' alt='ssss'/></a>
alt(right) + s = ś, alt(right) + shift + s = Ś <a href='https://postimg.cc/image/ubyrr29h9/' target='_blank'><img src='https://s14.postimg.cc/ubyrr29h9/krzywy_domek.gif' border='0' alt='krzywy_domek'/></a>

**Comment by RicoElectrico** [comment]
Bump, this is a breaking issue. People on the Polish forum complain about this.

**Comment by bhousel** [comment]
> Bump, this is a breaking issue. People on the Polish forum complain about this.

I'll tag it as "help wanted".  I think the people complaining are also in the best position to actually submit a pull request with a fix.

**Comment by maraf24** [comment]
This problem doesn't occur with latest version of Chrome. 
I tested some other browsers:
Vivaldi on Chromium 65 - "ś" not working.  Vivaldi on Chromium 68 - "ś" working.
Firefox 62beta - not working; Firefox Nightly - working.

I don't know what they changed.

**Comment by Dimitar5555** [comment]
It sounds like this issue might be fixed in the current browser versions. Can someone with Polish keyboard test and verify if it is fixed?
